static/js/shortcut_notification.js

```diff
@@ -0,0 +1,60 @@
+var shortcut_notification = (function () {
+
+var exports = {};
+
+exports.notify = (function () {
+    var meta = {
+        stream: null,
+        topic: null,
+        hide_me_time: null,
+        alert_hover_state: false,
+        $mute: null,
+    };
+    var animate = {
+        fadeOut: function () {
+            if (meta.$mute) {
+                meta.$mute.fadeOut(500).removeClass("show");
+            }
+        },
+        fadeIn: function () {
+            if (meta.$mute) {
+                meta.$mute.fadeIn(500).addClass("show");
+            }
+        },
+    };
+    setInterval(function () {
+        if (meta.hide_me_time < new Date().getTime() && !meta.alert_hover_state) {
+            animate.fadeOut();
+        }
+    }, 100);
+
+    return function () {
+        var $exit = $("#pressed_C_deprecated_notification .exit-me");
+
+        if (!meta.$mute) {
+            meta.$mute = $("#pressed_C_deprecated_notification");
```

Comment:
This entire module is dedicated to the shortcut notification? Hm, I'd try and improve it so that it's less specific so other modules could possibly use it in the future.